packages/master-detail-layout/src/vaadin-master-detail-layout.js

```diff
@@ -9,8 +9,118 @@ import { defineCustomElement } from '@vaadin/component-base/src/define.js';
 import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
 import { PolylitMixin } from '@vaadin/component-base/src/polylit-mixin.js';
 import { ResizeMixin } from '@vaadin/component-base/src/resize-mixin.js';
+import { addGlobalThemeStyles } from '@vaadin/vaadin-themable-mixin/register-styles.js';
 import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
+addGlobalThemeStyles(
```

Comment:
We could also use `SlotStylesMixin` like we do in some field components that add their styles globally.
This way it would also work for cases where `vaadin-master-detail-layout` is wrapped in shadow DOM.

Reply:
Refactored it to use `SlotStylesMixin`